README.md

```diff
@@ -24,6 +24,12 @@ directory:
 scim:
   create_email_identities: true
   create_role_groups: true
+  group_mappings:
+    - group: app-admin
+      type: group
```

Comment:
I think the example would be easier to understand if the object of the relation would be any type other than `group`. The subject of group mappings is always a `group` so I think using a different type as the object would make it easier to see what's what.